week7/cooper_client_react/11_client_user_authentication.md

```diff
@@ -258,53 +277,80 @@ Like we did with the `CooperCalculator`, we will extract the login function to a
 
 We will make a call to our backend API over the network and will need to add a library to be able to perform that call. The library we will use is [Axios](https://github.com/axios/axios). You install it like this:
 
-`$ npm i -S axios`
+`$ yarn add axios`
+
+And configure axios a little in our `index.js`:
+
+```js
+// src/index.js
+
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
+import axios from "axios";
+
+axios.defaults.baseURL = "http://localhost:3000/api/v1";
+
+ReactDOM.render(<App />, document.getElementById("root"));
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.unregister();
+```
 
 Let's add that module:
 
-`$ touch src/Modules/Auth.js`
+`$ touch src/modules/auth.js`
 
 Add this there:
-```
-import axios from 'axios'
 
-const apiUrl = 'http://localhost:3000/api/v1';
+```js
+import axios from "axios";
+
+const apiUrl = "http://localhost:3000/api/v1";
 
 const authenticate = async (email, password) => {
```

Comment:
see if you can refactor this method and the `storeAuthCredentials`
Feels like we can do it in a much simpler and dry way

Reply:
Done